Gemfile

```diff
@@ -86,6 +86,8 @@ gem 'aws-sdk-s3'
 gem 'meta-tags'
 gem 'mini_magick'
 
+gem 'counter_culture', '~> 2.0'
+
```

Comment:
Why do we need the gem?  vs. using just the standard Rails counter functionality ?

What does this accomplish  that can't be done with the standard Rails `counter` ?

Reply:
For this particular usage, there is no distinct advantage to using the gem, other than:

1. It auto-generates the migration file
2. It comes with a utility function that we can run periodically to ensure our counter cache values are correct

Both of the above are not difficult to do by themselves, but it's also code I didn't have to write.

In a larger sense, however, I was thinking that this is the first instance where we're introducing a counter cache - if so, lets set the pattern by using a tool that provides much more functionality in case we want to use that in the future, and otherwise _does not cost us anything more than the standard technique does_.

Reply:
Makes total sense.  
I also wanted to make sure I wasn't missing some functionality that was being used somewhere that I wasn't seeing.